src/components/navigation/header/config/drawerNodeItems.ts

```diff
@@ -739,6 +777,25 @@ export const drawerNodeItems = ({
             },
             brand: {
               autoscout24: true,
+              motoscout24: false,
+            },
+          },
+        },
+        {
+          translationKey: 'header.userMenu.reviews',
+          link: {
+            de: '/de/ip/autoscout24-3175-steffisburg/dealerrating?accountid={accountId}&tab=dealerrating',
```

Comment:
We will remove the rating feature for the PDP release but it can be tackled separately. However, I think this dealer name should not be hardcoded, no?

Reply:
I was not aware that is dealer name :( Any chance we can remove this navigation item now?

Reply:
yes remove it. We have it on our current sprint 👍